projects/storefrontstyles/scss/_versioning.scss

```diff
@@ -64,6 +64,81 @@ $useLatestStyles: false !default;
   }
 }
 
+@function getVersion($value) {
+  $major: 0;
+  $minor: 0;
+
+  $string: toString($value);
+  $i: str-index($string, '.');
+  @if $i != null {
+    $major: toNumber(str-slice($string, 1, $i - 1));
+    $minor: toNumber(str-slice($string, $i + 1));
+  } @else {
+    $major: toNumber($string);
+  }
+
+  @return (major: $major, minor: $minor);
+}
+
+@function toString($value) {
+  @if type-of($value) == 'number' {
+    @return #{$value};
+  } @else if type-of($value) == 'string' {
+    @return $value;
+  } @else if type-of($value) == 'list' {
+    @return toString(nth($value, 1));
+  } @else {
+    @error 'Input #{$value} is invalid';
+  }
+}
+
+@function toNumber($string) {
+  $strings: '0' '1' '2' '3' '4' '5' '6' '7' '8' '9';
+  $numbers: 0 1 2 3 4 5 6 7 8 9;
+
+  $result: 0;
+  @for $i from 1 through str-length($string) {
+    $character: str-slice($string, $i, $i);
+    $index: index($strings, $character);
+    $number: nth($numbers, $index);
```

Comment:
I think the index is already the number we want.  Maybe there is no need for `$numbers: 0 1 2 3 4 5 6 7 8 9;`

Reply:
index() starts from 1, not from 0. So, e.g. for character `5`, `index$` is 6. `nth` gets the 6th number from `$numbers`, so 5 is the result. Yes, we can get `$number` from `$index - 1`.